mempool/src/pool/feerate.rs

```diff
@@ -31,10 +31,15 @@ pub struct FeeRate {
 }
 
 impl FeeRate {
+    // TODO: remove this, use the more descriptive from_amount_per_kb instead.
```

Comment:
Why not kill this off immediately? Seems to be a copy of the function you wrote.